Add back auto-refresh on dashboard (addresses #2668) (#2761)
influxdata/docs-v2: content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md

> 

Reviewer: Should these changes be in both Cloud and OSS docs?

Reviewer 2: @ShmuelLotman is auto-refresh in Both Cloud and OSS?

Click the refresh button (**{{< icon "refresh" >}}**) to manually refresh the dashboard's data.

#### Manually refresh a single dashboard cell

1. Click the **{{< icon "gear" >}}** on the dashboard cell you want to refresh.
2. Click **{{< icon "refresh" >}} Refresh**.

### Automatically refresh dashboard

Click **Enable Auto Refresh**. In the window that appears, configure the following options:
  - **Until**: Select **Indefinite** to automatically refresh continuously. Select **Custom** to set a time to stop automatically refreshing.

>   - **Inactivity Timeout**: Set how long the dashboard should be inactive for to time out and stop automatically refreshing.

Reviewer: ```suggestion
  - **Inactivity Timeout**: Set how long the dashboard should be inactive before timeout stops automatically refreshing.
```

Reviewer: is this change accurate?

Reviewer 3: Maybe its worth saying "user should be inactive" or "page should be inactive" to be clearer about what is the trigger for the timeout?

Reviewer 2: How about `"Set how long the dashboard should be inactive before the auto-refresh times out and stops automatically refreshing" ` ?

Reviewer 3: "the dashboard should be inactive" is the part im not sure about whether the word should be dashboard or user

Reviewer 2: yeah sorry I didn't see your comment because I didn't refresh github
